SCClassLibrary/Common/GUI/tools/HelpBrowser.sc

```diff
@@ -307,6 +313,42 @@ HelpBrowser {
 				toggleFind.value;
 			};
 
+            if (key == keySlash ) {
```

Comment:
I would replace this long sequence of `if` with a `switch` or a `case` structure

Reply:
ok! Thank you @elgiano for checking and I apologize for my issues. Yes, `switch` would be better. I am going on it.

May I ask you a question? I think that my previous PR #5286 is now not obsolete and should be deleted, because I did this PR on top of it, so it includes same change in  [QWebView](https://github.com/supercollider/supercollider/pull/5297/commits/e183794fdbb0d70b66536272ca2f617ed908a23b)  class. In cpp also. Somehow I constructed working code, and it works, at least on my Arch :)

Am I right? Sould I delete #5286 ?

Thank you and Merry Christmas! For me very special this time ! For first time commiting here, and I am really glad that it seem that my code will go up ;) Hehe. Thank you for corrections.

paum
``

Reply:
Hi @paum3! 
It looks like #5286 contains a single commit, which is also included in this PR. And since they are different branches, it should be safe to delete #5286. 
And by the way, thanks for your contribution!

Reply:
OK. I just deleted it. Thank you for advice.  I will look on this PR soon. Merry xmas.